src/hdusd/mx_nodes/nodes/__init__.py

```diff
@@ -0,0 +1,58 @@
+# **********************************************************************
+# Copyright 2020 Advanced Micro Devices, Inc
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ********************************************************************
+import bpy
+import nodeitems_utils
+from nodeitems_utils import NodeCategory, NodeItem
+
+from ...utils import HDUSD_LIBS_DIR
+from .. import log
+from .base_node import create_node_types, MxNodeSocket
+
+
+node_types = create_node_types([
+    HDUSD_LIBS_DIR / "materialx/libraries/bxdf/standard_surface.mtlx",
+    # HDUSD_LIBS_DIR / "materialx/libraries/pbrlib/pbrlib_defs.mtlx",
```

Comment:
Why are these ones skipped?

Reply:
Maybe I did this.  https://github.com/materialx/MaterialX/blob/master/libraries/pbrlib/pbrlib_defs.mtlx but I think they're useful

Reply:
There are lot of nodes there and we need to think how to group them. This I plan to do in next task.
Also pbrlib_defs.mtlx has several interpretations of math nodes, for example 3 add nodes, 6 multiply nodes etc. Therefore I decided not to include it in this PR.
Also there are lot of node definitions in stdlib_defs.mtlx. This also will be included in next task.